app/assets/stylesheets/global.scss

```diff
@@ -7,29 +7,29 @@ body {
   font-family: $body_fonts;
 }
 
-h1, h2, h3, h4, h5, h6 {
-  font-family: $headline-fonts;
-}
-
-h1 {
-  color: $primary;
-  font-size: 2.25rem;
-  font-weight: 300;
-  letter-spacing: 0.4px;
-}
-
-h2 {
-  font-size: 1.5rem;
-  font-weight: 700;
-}
-
-h3 {
-  font-size: 1.5em;
-}
-
-h2, h3 {
-  color: $text-primary;
-}
+// h1, h2, h3, h4, h5, h6 {
```

Comment:
I think it would be better to just delete all the commented out styles now we have them over in the typography stylesheet 😄